include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -6688,6 +6688,71 @@
       "capabilities" : [ "SubgroupAvcMotionEstimationINTEL" ],
       "version" : "None"
     },
+    {
+      "opname" : "OpSubgroupAvcSicGetInterRawSadsINTEL",
+      "class"  : "@exclude",
+      "opcode" : 5816,
```

Comment:
It seems this created two `OpSubgroupAvcSicGetInterRawSadsINTEL`.  Is one supposed to be an alias, or we can have just one?

Reply:
I suppose it's an artifact appeared after rebase.